lib/neovim-plugin.nix

```diff
@@ -90,6 +90,21 @@ with lib;
         {
           enable = mkEnableOption originalName;
 
+          configLua = mkOption {
+            type = types.lines;
+            default = "";
+          };
+
+          configLuaPre = mkOption {
+            type = types.lines;
+            default = "";
+          };
+
+          configLuaPost = mkOption {
+            type = types.lines;
+            default = "";
+          };
```

Comment:
Do you think these should be hidden, until they will work as expected for _all_ plugins that have these options added?

Currently for many plugins these options will not actually do what they claim (add config beside the plugin's config) since the plugin implementation isn't updated to use the new options internally.

Reply:
Well I should have updated all plugins declared with mkNeovimPlugin! So it _should not_ be a lie

Reply:
Ok, if so then I'm fine with them being visible. The touched file count is suspiciously low, but I'll trust you :grin: